test/vending/VendingMachine.test.js

```diff
@@ -242,23 +322,23 @@ describe("VendingMachine", () => {
 
     context("when caller has no T tokens", () => {
       it("should revert", async () => {
-        const amount = 1
+        const amount = convertToT(to1e18(1)).result
         await tToken.connect(thirdParty).approve(vendingMachine.address, amount)
         await expect(
           vendingMachine.connect(thirdParty).unwrap(amount)
-        ).to.be.revertedWith("Transfer amount exceeds balance")
+        ).to.be.revertedWith("Can not unwrap more than previously wrapped")
       })
     })
 
     context("when token holder has not enough T tokens", () => {
       it("should revert", async () => {
-        const amount = tAmount.add(1)
+        const amount = tAmount.add(to1e18(42))
```

Comment:
Okay, now I started thinking there must be something special about 42 that I do not see yet... 🤔